wlhosted/templates/legal/documents/tos.html

```diff
@@ -82,44 +82,44 @@ <h2>{% blocktrans %}Liability for Damage{% endblocktrans %}</h2>
+<p class="item">{% blocktrans %}The Provider declares that it will collect Personal Data only in the extent necessary for meeting the above-specified purpose and process the Personal Data in accordance with the purpose for which it was collected. The Provider’s employees and other individuals who process the Personal Data on the basis of a contract with the Provider shall maintain confidentiality of the Personal Data, even after the termination of their employment or business relationship with the Provider.{% endblocktrans %}</p>
 
-<p class="item">{% blocktrans %}The Provider declares that he will process the Personal Data by electronic means (automatically) via computer programs for which the Provider has secured the right of use, in particular the Service.{% endblocktrans %}</p>
+<p class="item">{% blocktrans %}The Provider declares that it will process the Personal Data by electronic means (automatically) via computer programs for which the Provider has secured the right of use, in particular the Service.{% endblocktrans %}</p>
 
-<p class="item">{% blocktrans %}The User’s consent under this Article 5 of the Terms of Service constitutes free and informed juridical act which concerns the consent of the subject of the data (the User) with processing of the Personal Data.{% endblocktrans %}</p>
+<p class="item">{% blocktrans %}The User’s consent under this Article 5 of the Terms of Service constitutes a free and informed juridical act which concerns the consent of the subject of the data (the User) with processing of the Personal Data.{% endblocktrans %}</p>
 
 
 <h2>{% blocktrans %}Translations{% endblocktrans %}</h2>
 
-<p class="item">{% blocktrans %}The Service organizes translation into individual Projects, where the Owner is responsible for managing them and for specifying accurately the Project license.{% endblocktrans %}</p>
+<p class="item">{% blocktrans %}The Service organizes translation into individual Projects, where the Owner is responsible for managing them and for specifying accurately the license of the Project.{% endblocktrans %}</p>
 
-<p class="item">{% blocktrans %}Not specifying translation license means that the translations are available under same license as project itself.{% endblocktrans %}</p>
+<p class="item">{% blocktrans %}Not specifying translation license means that the translations are available under same license as the given Project itself.{% endblocktrans %}</p>
 
 <p class="item">{% blocktrans %}Should the Project opt in for the Translation Memory, license to use the translation is granted to all Translation Memory users.{% endblocktrans %}</p>
```

Comment:
Does the translation memory license extend to translation house and other related services?